### ISSUE TITLE
[Request] Arithmetic (Shift, Or, Etc) Operators 

### ISSUE THREAD

**Issue by terminalsin** [manifold-systems/manifold#537]
**Is your feature request related to a problem? Please describe.**
Hi! I use Manifold a lot for writing an IR application, and I've found myself using it specifically when writing expressions, ie:

**Without manifold**
```java
new ArithmeticExpr(new ConstantExpr(1), new ConstantExpr(2), ArithmeticOperator.PLUS)
``` 

**With manifold**
```java
cnst(1) + cnst(2)
```

Simplifying my IR mechanics a lot. However, I'm extremely saddened to see the lack of implementation of these operators:
- >>
- <<
- >>>
- |
- &
- ^

**Describe the solution you'd like**
If I understand correctly, it'd be quite easy to implement them by adding the tags via https://github.com/manifold-systems/manifold/blob/master/manifold-core-parent/manifold/src/main/java/manifold/internal/javac/ManAttr.java#L66

All would be needed is the exact same coverage as typical booleans and integers!

**Describe alternatives you've considered**
No alternatives considered, this is internal only.

**Additional context**
n/a

**Comment by terminalsin** [comment]
Perhaps merge https://github.com/manifold-systems/manifold/pull/458

**Comment by rsmckinney** [comment]
This feature is high on the list. I'll get to it soon.

**Comment by terminalsin** [comment]
> This feature is high on the list. I'll get to it soon.

God bless. Not to press but any eta? Could perhaps compiling a comprehensive PR accelerate this to production? Willing to make any sacrifice to help you move this forward 🫶

**Comment by rsmckinney** [comment]
I'm just finishing up an IJ plugin issue, putting that to bed now. This one is next. I like the determination! But I'll bang this out; should be on the shelf within a week. Let me know if your need for this is more pressing. Cheers.

**Comment by terminalsin** [comment]
> I'm just finishing up an IJ plugin issue, putting that to bed now. This one is next. I like the determination! But I'll bang this out; should be on the shelf within a week. Let me know if your need for this is more pressing. Cheers.

Absolute legend! Make sure to keep time for family and new years. Love the dedication to this awesome project, keep it up. You've transformed our workflow lives.

**Comment by rsmckinney** [comment]
Feature available with release 2024.1.1
